scipy/signal/ltisys.py

```diff
@@ -375,6 +376,10 @@ class lti(object):
     of one of its subclasses: `StateSpace`, `TransferFunction` or
     `ZerosPolesGain`.
 
+    If a `TransferFunction` is passed in for `*system`, coefficients for both
+    the numerator and denominator should be specified in descending exponent
+    order (e.g. s^2 + 3s + 5 would be represented as [1, 3, 5]).
+
```

Comment:
You aren't passing in a `TransferFunction` object: rather you are passing in numerator and denominator polynomials to *create* a `TransferFunction` object.